plugins/themes/default/styles/objects/article_details.less

```diff
@@ -39,8 +43,16 @@
 	}
 
 	.sub_item {
+
 		margin-bottom: @double;
 
+		.label {
+			margin: 0;
+			font-size: @font-sml;
+			font-weight: @normal;
+			color: @text-light;
+		}
+
```

Comment:
This block can be removed because these styles are overridden by the other `.label` styles.